grails-app/conf/RestApiUrlMappings.groovy

```diff
@@ -26,19 +26,20 @@
 class RestApiUrlMappings {
 
     // grails url-mappings-report can come handy here...
+    static String version = 'v1'
```

Comment:
Not sure if this should be in a variable, we want to be able to support multiple versions at the same time. So at least this shouldn't be named 'version'